.github/workflows/integration_test.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           docker ps -a
           mkdir -p /tmp/.kube
           kind get kubeconfig --name=kind > /tmp/.kube/config
-          kubectl port-forward --address localhost $(kubectl -n kepler get pods -o name) 9102:9102 -n kepler -v7 &
+          while true; do kubectl port-forward --address localhost -n kepler service/kepler-exporter 9102:9102; done &
```

Comment:
do we need a while true here?

Reply:
Yes, because when it loose the connect, it immediately creates a new one.